locale/Makefile

```diff
@@ -44,33 +41,20 @@ uniq-po:
 		msguniq $$f -o $$f ; \
 	done
 
-tx-pull:
+tx-pull: $(EDITFILES)
 	tx pull -f
-	for f in $(POFILES) ; do \
+	for f in $(EDITFILES) ; do \
 		sed -i 's/^\("Project-Id-Version: \).*$$/\1$(DOMAIN) $(VERSION)\\n"/' $$f; \
 	done
-	-git commit -a -m "i18n - pulling from tx"
-
-reset-po:
-	# merging po files is unnecessary when using transifex.com
-	git checkout -- ../locale/*/*po
-
-tx-update: tx-pull reset-po $(MOFILES)
-	# amend mo files
-	git add ../locale/*/LC_MESSAGES
-	git commit -a --amend -m "i18n - pulling from tx"
-	-echo Changes commited!
 
-extract:
-	rxgettext \
-		--sort-output \
-		--sort-by-msgid \
-		--no-wrap \
-		--no-location \
-		-o ${DOMAIN}.pot \
-		--package-name=${DOMAIN} \
-		--package-version="${VERSION}" \
-		--msgid-bugs-address=foreman-dev@googlegroups.com \
-		--copyright-holder="Foreman developers" \
-		--copyright-year=$(shell date +%Y) \
-		$(shell find ../app -type f -name *.rb -o -name *.erb)
```

Comment:
Can you keep this target for the future? It looks like the rake task is unstable, I'd like to keep it handy when it does not work so release is not blocked. You can rename it to `external-extract` or something like that, maybe add a coment why we have it here.